src/state/application/reducer.test.ts

```diff
@@ -14,6 +14,7 @@ describe('application reducer', () => {
 
   beforeEach(() => {
     store = createStore(reducer, {
+      fiatOnramp: { available: false, availabilityChecked: false },
```

Comment:
what changed to cause the linter to catch this now?

Reply:
typechecking was being done as part of react-scripts, which seems to have only typechecked source files (not test files).
by breaking typechecking into its own step, it's applied to all files.